microprofile/opentracing-smallrye/src/main/java/org/wildfly/extension/microprofile/opentracing/smallrye/TracingDeploymentProcessor.java

```diff
@@ -0,0 +1,124 @@
+package org.wildfly.extension.microprofile.opentracing.smallrye;
+
+import io.smallrye.opentracing.SmallRyeTracingDynamicFeature;
+import org.jboss.as.ee.weld.WeldDeploymentMarker;
+import org.jboss.as.server.deployment.DeploymentPhaseContext;
+import org.jboss.as.server.deployment.DeploymentUnit;
+import org.jboss.as.server.deployment.DeploymentUnitProcessor;
+import org.jboss.as.server.deployment.Phase;
+import org.jboss.as.web.common.WarMetaData;
+import org.jboss.as.weld.deployment.WeldPortableExtensions;
+import org.jboss.metadata.javaee.spec.ParamValueMetaData;
+import org.jboss.metadata.web.jboss.JBossWebMetaData;
+import org.jboss.metadata.web.spec.ListenerMetaData;
+import org.wildfly.extension.microprofile.opentracing.smallrye.deployment.TracerInitializer;
+import org.wildfly.extension.microprofile.opentracing.smallrye.deployment.TracingCDIExtension;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TracingDeploymentProcessor implements DeploymentUnitProcessor {
+  public static final Phase PHASE = Phase.POST_MODULE;
+  public static final int PRIORITY = 0x4000;
```

Comment:
Same comment about the Phase and the discussion on https://github.com/wildfly/wildfly/pull/11320